hooks/playbooks/compute-iscsi-config.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Increase quotas for the service project
+  hosts: computes
+  gather_facts: false
+  tasks:
+    - name: Change timeout in iscsid.conf
+      become: true
+      ansible.builtin.lineinfile:
+        path: /etc/iscsi/iscsid.conf
+        line: "{{ item }}"
+      loop:
+        - 'node.session.initial_login_retry_max = 3'
```

Comment:
I think we can change this to 1, because it either connects or it doesn't, and these are for tests, not production environments...